chapters/04-function-development.Rmd

```diff
@@ -13,10 +13,36 @@
 - Function documentation (with roxygen2), part 1
 - To use `@examples` to help with creating function
 
-
-
 ## Final exercise
 
 <!-- NOTE: Preferably something linked to working towards completing the project assignments, maybe to be used as a milestone? -->
 
+1. Add a `FIXME()` function to your FIXME package
```

Comment:
We may need to make this a bit more generic given learners will have different assignment packages. For now its a good starting point.